### ISSUE TITLE
App is blank (can't open project or create new project)

### ISSUE THREAD

**Issue by 54mu3l** [HaikuTeam/animator#1085]
Hi there!

I installed [Haiku-5.1.2.dmg](https://github.com/HaikuTeam/animator/releases/download/5.1.2/Haiku-5.1.2.dmg) on my Mac.

But I always get a blank (grey/black) screen and nothing is shown. Here a screenshot:
<img width="1411" alt="Screenshot" src="https://user-images.githubusercontent.com/17022670/168243676-87d77d61-6857-40f8-a2f4-bb8134142f57.png">

Doesn't matter if I choose to select an existing project or to create a new project.

Also the menu is disabled:
<img width="621" alt="image" src="https://user-images.githubusercontent.com/17022670/168244046-5d0c3e25-d576-4ea4-bfa7-fbefa79a8eea.png">

Is it working for you on MacOS?
Any idea what this could be?

Any help would be greatly appreciated!
Thanks.

___
ps: I'm still on MacOS Big Sur

**Comment by adenirantayo** [comment]
Were you connected to the internet when you opened Haiku? It needs to be connected to the internet

**Comment by MrDorff** [comment]
Have the same issue here. Computer well connected with internet.

**Comment by MrDorff** [comment]
Console said that a Python Lib is missing. But not better after installing one… It works on a other machine; but not on those one… very frustrating

**Comment by adenirantayo** [comment]
I can't run Haiku as well, the issue is that I do get a popup message insisting that I should connect to the internet. I have reported more than a month ago twice.
![166184261-bd8fa45b-ceba-49ae-8cc2-7cc8da4333c1 jpeg-1](https://user-images.githubusercontent.com/25295202/169271031-54f529f5-84b0-4c46-9b4f-0dbff58542b7.jpg)
I think it's a known problem amongst the developers

**Comment by 54mu3l** [comment]
> Were you connected to the internet when you opened Haiku? It needs to be connected to the internet

Yes I'm connected to the internet.


> I can't run Haiku as well, the issue is that I do get a popup message insisting that I should connect to the internet.

I don't get any messages.

**Comment by Babouchhhe** [comment]
Hi @54mu3l , 

I have the exactly same issue with Haiku.
Did you find a solution?

Thanks!

**Comment by adenirantayo** [comment]
No solution that I know of, although you could try Glaxnimate instead. Although it's UI isn't like Haiku, but it has more export options

**Comment by drucinsky** [comment]
@adenirantayo  Glaxnimate has export to Lottie file?
Okay i read documentation - Features section and all see. 
I must try use it because Haiku dont working on windows :( 
Maybe after i try run it on wsl Ubuntu.

**Comment by adenirantayo** [comment]
Yes, Glaxnimate has lottie export. It has mp4 and many more export options. You can copy(click cltr+c) a shape in inkscape and paste (cltr+v) directly in Glaxnimate
